content/cloud/cli-reference.md

```diff
@@ -475,11 +557,12 @@ OPTIONS:
 
 ### Up
 
+Applications without triggers will accept any of the following options. The trigger-specific options are over and above what is shown directly below. Please see the next section for additional trigger options.
```

Comment:
Sorry, I explained this badly.  There are no applications without triggers, and `spin up` won't accept _any_ options without an application.  (But we have to do _something_ when a user types `spin up --help` without an application, which is where the confusion arises.)  It's more like "The following options are always available", _plus_ any trigger-specific options if an application is present.